torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -3020,6 +3020,40 @@ RegisterOperators reg2({
           return 0;
         },
         aliasAnalysisFromSchema()),
+    Operator(
+        "aten::all(int[] self) -> bool",
+        [](Stack& stack) {
+          c10::List<int64_t> l = pop(stack).toIntList();
+          push(stack, std::move(l));
+          for(int i = 0; i < l.size(); i++){
+            if(l[i] == 0){
+              push(stack, false);
+              break;
+            }
+            else{
+              push(stack, true);
+            }
+          }
+          return 0;
+        },
+        aliasAnalysisFromSchema()),
+    Operator(
+        "aten::all(bool[] self) -> bool",
+        [](Stack& stack) {
+          c10::List<bool> l = pop(stack).toBoolList();
+          push(stack, std::move(l));
+          for(int i = 0; i < l.size(); i++){
```

Comment:
nit: generally safer to use `int64_t`, although it is unlikely that we'll have a `2^32` long list

Reply:
I may come back to this down the track if we run into any problems.

Reply:
Why not just 
```
(const auto elem: l) . {
    if (!elem)
}
```

the indexing is unnecessary and iteration counter is unnecessary